packages/custom-url/src/Infrastructure/Sulu/Admin/CustomUrlAdmin.php

```diff
@@ -131,4 +132,18 @@ private function hasSomeWebspaceCustomUrlPermission(): bool
 
         return false;
     }
+
+    private function hasAnyWebspaceWithCustomUrls(): bool
+    {
+        foreach ($this->webspaceManager->getWebspaceCollection()->getWebspaces() as $webspace) {
+            foreach ($webspace->getPortals() as $portal) {
+                $environment = $portal->getEnvironment($this->environment);
+                if ($environment && \count($environment->getCustomUrls()) > 0) {
```

Comment:
```suggestion
                if (\count($environment->getCustomUrls()) > 0) {
```

Throws exception when not exists:

https://github.com/sulu/sulu/blob/e312b5e8544111d01e773df8b588135a703f6c42/src/Sulu/Component/Webspace/Portal.php#L221-L223